### PR TITLE
Add PowerBIDedicated schema

### PR DESCRIPTION
**Pull request by selasagi**
Add schema for new power bi azure resource

### CODE REVIEW

**Comment by selasagi** [comment]
Please review. Thanks

File: schemas/2017-10-01/Microsoft.PowerBIDedicated.json [hunk 1]
```diff
@@ -0,0 +1,94 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {
```
Comment (TianoMS) [inline schemas/2017-10-01/Microsoft.PowerBIDedicated.json]: >definitions [](start = 3, length = 11)

we have a practice in the order of the these top level properties. can you please follow that?

File: schemas/2017-10-01/Microsoft.PowerBIDedicated.json [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        },
+        "location": {
+          "type": "string",
+          "description": "Capacity location"
+        },
+        "tags": {
+          "oneOf": [
+            {
+              "type": "object",
+              "additionalProperties": { "type": "string" }
```
Comment (TianoMS) [inline schemas/2017-10-01/Microsoft.PowerBIDedicated.json]: >"additionalProperties": { "type": "string" } [](start = 14, length = 44)

Also, is it possible to format the json elements to be consistent with other files in this repo?

**Comment by Tiano2017** [comment]
PR looks good. Just two issues regarding format.

File: schemas/2017-10-01/Microsoft.PowerBIDedicated.json [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ]
+        }
+      },
+      "required": [
+        "type",
+	"apiVersion",
```
Comment (TianoMS) [inline schemas/2017-10-01/Microsoft.PowerBIDedicated.json]: >	 [](start = 0, length = 1)

please use spaces instead of tabs.

**Comment by selasagi** [comment]
tabs removed. thanks

**Pull request merged**